.github/workflows/build.yml

```diff
@@ -3,7 +3,9 @@ name: build
 on:
   push:
     # Sequence of patterns matched against refs/heads
-    branches: [ main ]
+    branches:
+      # Push events on the main branch
+      - main
 
```

Comment:
This is where there is a conflict. It seems easy enough to fix to just remove this from the changeset.